### ISSUE TITLE
Native Notebooks - Global Status Bar Items (order switching)

### ISSUE THREAD

**Issue by claudiaregio** [microsoft/vscode-jupyter#5839]
Sometimes the button to connect to remote jupyter server and the kernel are switched in places, let's make sure these stay in the same order 

![image](https://user-images.githubusercontent.com/19276747/117872894-0ec94300-b254-11eb-86e2-50ac7a18feed.png)

**Comment by DonJayamanne** [comment]
@claudiaregio this was brought up when we added the status bar, we can't control this, its a best effort.

**Comment by DonJayamanne** [comment]
I personally think we should just fix our remote experience, rather than having users to be either on local only or remote only (& have to reload vscode when switching between the two). Else all of this just feels like constant bandaids to me..

**Comment by claudiaregio** [comment]
@DonJayamanne agreed, since that will come a bit further down the line we can at least do this in the meantime to try to have a consistent experience and not mess with muscle memory of users who have learned the location of kernel picker + remote server entry point

**Comment by claudiaregio** [comment]
Change was pushed regarding this issue that may help.  https://github.com/microsoft/vscode/commit/381e7c2d0bc44646a92d212b15a41f4214b781c7
